### PR TITLE
Rename velox backendsapi to make consistent with CH

### PR DESCRIPTION
**Pull request by yma11**
## What changes were proposed in this pull request?
Rename velox backendsapi to make consistent with CH and make it easier to distinguish it's Velox's impl

## How was this patch tested?

N/A

### CODE REVIEW

**Comment by yma11** [comment]
@zhztheplayer @zhouyuan can you help review this PR? Thanks.

File: backends-velox/src/main/resources/META-INF/services/org.apache.gluten.backendsapi.Backend [hunk 1]
```diff
@@ -1 +1 @@
-org.apache.gluten.backendsapi.velox.VeloxBackend
+org.apache.gluten.backendsapi.VeloxBackend
```
Comment (zhztheplayer) [inline backends-velox/src/main/resources/META-INF/services/org.apache.gluten.backendsapi.Backend:1]: I see CH backend's impl is located at package `org.apache.gluten.backendsapi.clickhouse`. Should we do such movement only for Velox backend?

https://github.com/apache/incubator-gluten/blob/972004395369224e572690b0fe53d90c19cd3a94/backends-clickhouse/src/main/resources/META-INF/services/org.apache.gluten.backendsapi.Backend#L1C1-L1C51

Comment (zhztheplayer) [reply backends-velox/src/main/resources/META-INF/services/org.apache.gluten.backendsapi.Backend:1]: Also, what's the point of doing the movement?

**Review by zhztheplayer — COMMENTED** [review]
Other changes LGTM. Thanks!

File: backends-velox/src/main/resources/META-INF/services/org.apache.gluten.backendsapi.Backend (hunk 1 above)
Comment (yma11) [reply backends-velox/src/main/resources/META-INF/services/org.apache.gluten.backendsapi.Backend:1]: I only want to change the API names but not include this folder change. Let me recover it back.

**Review by zhztheplayer — APPROVED** [review]
Thanks!

**Pull request merged**